css/style.css

```diff
@@ -0,0 +1,123 @@
+*{
```

Comment:
you can also use flex box flex-wrap to make the screen flexable, we usually use flex box and grid instead of media